src/ui/DynamicFacet/DynamicFacetRange.ts

```diff
@@ -57,6 +57,12 @@ export class DynamicFacetRange extends DynamicFacet implements IComponentBinding
      * @examples date
      */
     valueFormat: ComponentOptions.buildStringOption({ defaultValue: DynamicFacetRangeValueFormat.number, section: 'CommonOptions' }),
+    /**
+     * The currency symbol used if the [`valueFormat`]{@link DynamicFacetRangeValueFormat.options.valueFormat} is [`currency`]{@link DynamicFacetRangeValueFormat.currency}.
+     *
+     * **Default:** `$`
```

Comment:
If the default remains `$`, then there is no need to include it in the description; TypeDoc will generate a tag for the default value.

However, if you decide to apply @olamothe 's suggestion and try to detect the current locale, we will need to properly document this default behavior. Suggestion:

> By default, the component uses the currency associated with the currently loaded culture file (see [Changing the Language of Your Search Interface](https://docs.coveo.com/421/)).

Reply:
Yes I will just remove that option

Reply:
Mmm I don't know if we should remove it altogether. For instance, we have a fr-FR culture file, but we don't have a fr-CA one, so "€" could end up being used as a currency symbol even for French Canadian end-users.

Reply:
Understood!